Content.Server/Zombies/PendingZombieComponent.cs

```diff
@@ -35,24 +40,22 @@ public sealed class PendingZombieComponent : Component
     [ViewVariables(VVAccess.ReadWrite), DataField("maxInfectionLength")]
     public float MaxInfectionLength = 120f;
 
-    /// <summary>
-    /// Infection warnings are shown as popups, times are in seconds.
-    ///   -ve times shown to initial zombies (once timer counts from -ve to 0 the infection starts)
-    ///   +ve warnings are in seconds after being bitten
-    /// </summary>
-    [DataField("infectionWarnings")]
-    public Dictionary<int, string> InfectionWarnings = new()
-    {
-        {-45, "zombie-infection-warning"},
-        {-30, "zombie-infection-warning"},
-        {10, "zombie-infection-underway"},
-        {25, "zombie-infection-underway"},
-    };
-
     /// <summary>
     /// A minimum multiplier applied to Damage once you are in crit to get you dead and ready for your next life
     ///   as fast as possible.
     /// </summary>
     [DataField("minimumCritMultiplier")]
     public float MinimumCritMultiplier = 10;
+
+    /// <summary>
+    ///    When Initial Infected can first turn (copied from ZombieRuleComponent)
+    /// </summary>
+    [DataField("firstTurnAllowed"), ViewVariables(VVAccess.ReadWrite)]
+    public TimeSpan FirstTurnAllowed = TimeSpan.Zero;
```

Comment:
As above, also missing customtypeserializer and pause support.

Reply:
How would we even put pause support in this? Wouldn't that require us to move all the times into the future while paused?